frame.go

```diff
@@ -0,0 +1,70 @@
+package errtrace
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// Frame is a single frame in an error trace
+// identifying a site where an error was wrapped.
+type Frame struct {
+	// Func is the fully qualified function name
+	// inside which the error was wrapped.
+	Func string
```

Comment:
optional: should we name this `Function` to align with `runtime.Frame`, and document it as the same?